ArcGISRuntimeSDKQt_CppSamples/Scenes/FeatureLayerExtrusion/FeatureLayerExtrusion.cpp

```diff
@@ -0,0 +1,116 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#include "FeatureLayerExtrusion.h"
+
+#include "FeatureLayer.h"
+#include "ServiceFeatureTable.h"
+#include "ArcGISTiledElevationSource.h"
+#include "Scene.h"
+#include "SceneQuickView.h"
+#include "OrbitLocationCameraController.h"
+#include "Point.h"
+#include "SimpleLineSymbol.h"
+#include "SimpleFillSymbol.h"
+#include "SimpleRenderer.h"
+#include "RendererSceneProperties.h"
+
+
+using namespace Esri::ArcGISRuntime;
+
+
+
+FeatureLayerExtrusion::FeatureLayerExtrusion(QQuickItem* parent /* = nullptr */):
+  QQuickItem(parent)
+{
+}
+
+void FeatureLayerExtrusion::init()
+{
+  // Register classes for QML
+  qmlRegisterType<SceneQuickView>("Esri.Samples", 1, 0, "SceneView");
+  qmlRegisterType<FeatureLayerExtrusion>("Esri.Samples", 1, 0, "FeatureLayerExtrusionSample");
+}
+
+void FeatureLayerExtrusion::componentComplete()
+{
+  QQuickItem::componentComplete();
+
+  // Create the feature service to use
+  m_featureTable = new ServiceFeatureTable(QUrl("http://sampleserver6.arcgisonline.com/arcgis/rest/services/Census/MapServer/3"), this);
+
+  // add the service feature table to a feature layer
+  m_featureLayer = new FeatureLayer(m_featureTable, this);
+
+  // set the feature layer to render dynamically to allow extrusion
+  m_featureLayer->setRenderingMode(FeatureRenderingMode::Dynamic);
+
+  // define line and fill symbols for a simple renderer
+  m_lineSymbol = new SimpleLineSymbol(SimpleLineSymbolStyle::Solid, QColor("Black"), 1.0f, this);
+  m_fillSymbol = new SimpleFillSymbol(SimpleFillSymbolStyle::Solid, QColor("Blue"), m_lineSymbol, this);
+  m_renderer = new SimpleRenderer(m_fillSymbol, this);
+
+  // set renderer extrusion mode to base height, which includes base height of each vertex in calculating z values
+  RendererSceneProperties props = m_renderer->sceneProperties();
+  props.setExtrusionMode(ExtrusionMode::BaseHeight);
+  m_renderer->setSceneProperties(props);
+
+  // set the simple renderer to the feature layer
+  m_featureLayer->setRenderer(m_renderer);
+
+  // Create a scene and give it to the SceneView
+  m_sceneView = findChild<SceneQuickView*>("sceneView");
+  Scene* scene = new Scene(Basemap::imagery(this), this);
+  Surface* surface = new Surface(this);
+  surface->elevationSources()->append(
+        new ArcGISTiledElevationSource(
+          QUrl("http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"),
+          this));
+  scene->setBaseSurface(surface);
+  scene->operationalLayers()->append(m_featureLayer);
+  m_sceneView->setArcGISScene(scene);
+
+
+  const Esri::ArcGISRuntime::Point m_lookAtPoint = Point(-10974490, 4814376, 0, SpatialReference::webMercator());
```

Comment:
don't need to scope with namespace since you put `using namespace Esri::ArcGISRuntime;` at the top. This can change to `const Point m_lookAtPoint...`. There are a few instances of this in the PR